src/prng.rs

```diff
@@ -275,7 +280,7 @@ mod tests {
             Prng::from_seed_stream(XofTurboShake128::seed_stream(&seed, b"", b""));
         let want_buffer = want.buffer.clone();
 
-        let got: Prng<FieldPrio2, _> = want.into_new_field();
+        let got: Prng<Field64, _> = want.into_new_field();
```

Comment:
I think this should use a field other than Field64, to maintain the test's semantics.

Reply:
Ugh !!! Missed that.

Replaced with `Field128`, and I renamed the test to make the semantics a little more clear.